mypy/messages.py

```diff
@@ -855,6 +855,11 @@ def unsupported_type_type(self, item: Type, context: Context) -> None:
     def redundant_cast(self, typ: Type, context: Context) -> None:
         self.note('Redundant cast to {}'.format(self.format(typ)), context)
 
+    def unimported_type_becomes_any(self, prefix: str, typ: Type, ctx: Context) -> None:
+        self.fail("{} becomes {} due to an unfollowed import (such imports occur either "
```

Comment:
I'm not sure it is good to have an error message that long. 

We could remove the explanation in parens but I'm not sure that most users will understand what an `unfollowed import` is otherwise.

@ddfisher what do you think?